docs/changelogs/v3.5.4.md

```diff
@@ -0,0 +1,91 @@
+**Feature or improvement description**
+Pull request to merge `rc-3.5.4` into `main` and create a tagged release for v3.5.4.
+
+See the milestone and project pages for additional information
+
+    https://github.com/OpenFAST/openfast/milestone/14
+
+Test results, if applicable
+See GitHub Actions
+
+### Release checklist:
+- [ ] Update the documentation version in docs/conf.py
+- [ ] Update the versions in docs/source/user/api_change.rst
+- [ ] Verify readthedocs builds correctly
+- [ ] Create a tag in OpenFAST
+- [ ] Create a merge commit in r-test and add a corresponding annotated tag
+- [ ] Compile executables for Windows builds
+    - [ ] AeroDyn_Driver_x64.exe
+    - [ ] AeroDyn_Driver_x64_OpenMP.exe
+    - [ ] AeroDyn_Inflow_C_Binding_x64.dll
+    - [ ] AeroDyn_Inflow_C_Binding_x64_OpenMP.dll
+    - [ ] BeamDyn_Driver_x64.exe
+    - [ ] DISCON.dll (x64)
+    - [ ] DISCON_ITIBarge.dll (x64)
+    - [ ] DISCON_OC3Hywind.dll (x64)
+    - [ ] DISCON_SC.dll (x64)
+    - [ ] FAST.Farm_x64.exe
+    - [ ] FAST.Farm_x64_OMP.exe
+    - [ ] FAST_SFunc.mexw64
+    - [ ] HydroDynDriver_x64.exe
+    - [ ] HydroDyn_C_Binding_x64.dll
+    - [ ] IfW_C_Binding_x64.dll
+    - [ ] InflowWind_Driver_x64.exe
+    - [ ] InflowWind_Driver_x64_OpenMP.exe
+    - [ ] MoorDyn_Driver_x64.exe
+    - [ ] MoorDyn_C_Binding_x64.dll
+    - [ ] OpenFAST-Simulink_x64.dll
+    - [ ] openfast_x64.exe
+    - [ ] Turbsim_x64.exe
+
+# Changelog
+
+## Overview
+
+This release includes performance improvements for BeamDyn (up to 30% speed increase), and a fix for stack overflows with FAST.Farm (Intel compilation).  A few other minor pdates are inluded as outlined below.
+
+Anyone using BeamDyn will want to update to this version.
+
+
+## General
```

Comment:
Seems like a lot of the file changes are tied to Python readers/writers for OpenFAST, which I don't see called out in these release notes.  I also see other bug fixes included in this release (e.g., HydroDyn), which I assume should be mentioned.

Reply:
You are right.  I'm probably missing a PR somewhere in the list.

Reply:
Actually, I'm missing two PR's. I'll get that updated.

Reply:
Updated.